monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+El proceso para Monero de minería prueba de trabajo (PoW) RandomX, arranca al ejecutarse un programa pseudoaleatorio (función de la nueva transacción), un nonce (número criptográfico arbitrario de un solo uso), una dirección de pago y datos del bloque anterior. Ejecutar un programa aleatorio como este es fácil para los ordenadores CPU, pero difícil para los ASIC. Luego, a la salida del programa, se le aplica una función hash criptográfica y convertida a un número de 256 bits. Si este número es menor en un umbral específico, cuenta como el haber minado un nuevo bloque.
 
-A beauty of this hashing process is that the payment address is specified before the hash is generated, which allows hashes to be created specifically for someone else. This property allows mining pools to function—hashes are made using the pool’s payment address. There are few analogies to a mining hash in the physical world. It is a creation that can have value only for the owner of the destination address, and its value only lasts until the next block is mined. Monero blocks are mined about every two minutes.
+Lo bueno de este proceso de hashing es que la dirección de pago está especificada antes de la generación del hash, lo que permite crear hashes especificamente para alguien más. Los hashes, se crean utilizando la dirección de pago, y esta propiedad, es la que le permite a los pools de minería funcionar. Hay ciertas analogías a la minería de hashes en el mundo real. Es una creación que solo tiene valor al dueño de la dirección de destino y dura hasta que el siguiente bloque sea minado. Los bloques de Monero son minados cada dos minutos.
 
-For RPC-Pay, the hashes are calculated by the RPC client itself or by a (more powerful) helper to the client, and the hashes are sent to the server as they are calculated. Every hash sent counts as payment. A miniscule percentage of the hashes will actually have value to the server—those that, when represented as a number, mine a block by being small enough. These successful hashes occur at random within the many payment hashes. The server, then, counts all the hashes for credit, but only receives payment itself when it receives and publishes a successful mining hash on the Monero network.
+Para RPC-Pay, los hashes son calculados por el cliente RPC mismo o por un ayudante al cliente (con mayor potencia), y los hashes son enviados al servidor a medida que son calculados. Cada hash enviado cuenta como pago. Un porcentaje minúsculo de los hashes que, cuando se representen como un número y minen un bloque por ser lo suficientemente pequeños, eventualmente tendrán valor al servidor. Estos hashes exitosos ocurren de manera aleatoria dentro de los tantos pagos con los hashes. El servidor, luego, cuenta todos los hashes para crédito, pero solo recibe pago a si mismo cuando recibe y publica un hash exitoso en la red de Monero.
```

Comment:
So far so good, just a small correction, as per [this](https://www.rae.es/dpd/asimismo):

_pero solo recibe pago a **sí** mismo cuando recibe y publica un hash exitoso en la red de Monero._

Don't remember if nowadays it's the same, some accents have been omitted, but just in case.

Reply:
Added the accent, read about it and it can be used.